.eslintrc.cjs

```diff
@@ -33,7 +33,7 @@ module.exports = {
   'rules': {
     'react/display-name': 'off',
     'no-unused-vars': [0],
-    'indent': ['error', 2],
+    'indent': ['error', 2, {'SwitchCase': 1}],
```

Comment:
Added to fix issue with index for switch/case
<img width="584" alt="Screenshot 2024-06-10 at 14 30 15" src="https://github.com/reaviz/reablocks/assets/5399389/0dfbfb51-cce7-48a5-935c-d5591e073dbc">